calculatorApi1.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+app.use(cors());
+const port = 3000;
+
+const add = (a, b) => {
+    let sum = a + b
+    return parseInt(sum);
+}
```

Comment:
No need to parseInt your return value. The sum should already be a number because you later make sure that the arguments passed are numbers.